qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -31,8 +31,9 @@ compatibility_test() {
         remove_existing_stackrox_resources
         setup_default_TLS_certs
 
-        deploy_stackrox
+        deploy_stackrox_with_custom_sensor "${SENSOR_CHART_VERSION}"
         echo "Stackrox deployed"
+        kubectl -n stackrox get deploy,ds -o wide
```

Comment:
Don't forget to remove this line before merging

Reply:
I think we should keep it. It doesn't harm to output the actual images in the cluster and it might help catch some problems in case we're deploying incorrect versions of collector or scanner.